INSTALL.md

```diff
@@ -26,6 +26,9 @@ You can link OpenTelemetry C++ SDK with libraries provided in [dependencies.md](
   the unittests. We use GoogleTest version 1.10.0 in our build system. To
   install GoogleTest, consult the [GoogleTest Build
   Instructions](https://github.com/google/googletest/blob/master/googletest/README.md#generic-build-instructions).
+- [Google Benchmark](https://github.com/google/benchmark) framework to build and run
+  benchmark tests. We use Benchmark version 1.5.3 or above. To install Benchmark,
```

Comment:
While you are adding benchmark as a pre-requisite, should we add a relative reference to [third_party_release](https://github.com/open-telemetry/opentelemetry-cpp/blob/main/third_party_release) to refer to the versions used for googletest and benchmark? This will ensure that minimal changes are required across documents whenever a given version is upgraded.

Reply:
```suggestion
  benchmark tests. We use Benchmark version 1.5.3 or above. To install Benchmark,
```
Currently I could see v1.5.3 is used for benchmark in the third party release. I have installed the latest version of benchmark v1.6.0 and it works fine. 
Coming to the GoogleTest version, the documentation refers to use 1.10.0 as pre requisites for Cmake, which is different from [third_party_release](https://github.com/open-telemetry/opentelemetry-cpp/blob/main/third_party_release) as it refers to `release-1.8.0-2523-ga6dfd3ac`. builds with 1.10.0 are successful.

I think the relative reference to be added, would like to listen from you if I am missing things that I need to consider in adding as relative reference.

Reply:
I think it should be fine to mention the versions used in the CI pipeline. With relative reference, I actually meant if we can provide the link to the [third_party_release](https://github.com/open-telemetry/opentelemetry-cpp/blob/main/third_party_release) doc instead of mentioning the version here.

- [GoogleTest](https://github.com/google/googletest) framework to build and run
  the unittests. Refer [third_party_release](https://github.com/open-telemetry/opentelemetry-cpp/blob/main/third_party_release)  for version of GoogleTest used in CI. To
  install GoogleTest, consult the [GoogleTest Build
  Instructions](https://github.com/google/googletest/blob/master/googletest/README.md#generic-build-instructions).
- [Google Benchmark](https://github.com/google/benchmark) framework to build and run
  benchmark tests. Refer [third_party_release](https://github.com/open-telemetry/opentelemetry-cpp/blob/main/third_party_release)  for version of Benchmark used in CI. To install Benchmark,
  consult the [GoogleBenchmark Build Instructions](https://github.com/google/benchmark#installation)


```

Reply:
sure, i totally agree with your point.